src/common/layout/Header.js

```diff
@@ -117,22 +118,25 @@ type Props = {
 
 const renderLoggedIn = (comesFromSearch: boolean) => {
   return (
-    <div className="loggedIn">
-      <div className="mobileOnly">
-        <BackButton text={comesFromSearch ? 'Search' : 'Back'} />
-      </div>
-      <div className="helpHeader">Help</div>
-      <div className="links">
-        <div className="faqLink desktopOnly">
-          <FullFAQLink className="primary" />
+    <React.Fragment>
+      <div className="loggedIn">
+        <div className="mobileOnly">
+          <BackButton text={comesFromSearch ? 'Search' : 'Back'} />
         </div>
-        <div className="desktopOnly">
-          <SignOutButton />
+        <div className="helpHeader">Help</div>
+        <div className="links">
+          <div className="faqLink desktopOnly">
+            <FullFAQLink className="primary" />
+          </div>
+          <div className="desktopOnly">
+            <SignOutButton />
+          </div>
         </div>
+        <style jsx>{loggedInStyle}</style>
+        <style jsx>{responsiveStyleHelperClasses}</style>
       </div>
-      <style jsx>{loggedInStyle}</style>
-      <style jsx>{responsiveStyleHelperClasses}</style>
-    </div>
+      <MobileBookingHeader />
```

Comment:
`display: none` on desktop is not enough - query is still sent

the same applies to regular booking page for desktop - `display: none` is not enough

Reply:
Do you have a suggestion to resolve this in the scope of this pull request?

Reply:
not particularly, google suggested me something like this: https://github.com/contra/react-responsive it would be fresh new dependency, but it's relatively small and imho better way than imperative `isMobile ? renderMobile() : render()`

Reply:
Seems pretty cool

Reply:
Also seems to work well :-)